src/models/util/ParseCalChartUtils.ts

```diff
@@ -0,0 +1,93 @@
+export function ReadFourCharCode(buffer: DataView, offset: number): string {
+  return String.fromCharCode(
+    buffer.getUint8(offset + 0),
+    buffer.getUint8(offset + 1),
+    buffer.getUint8(offset + 2),
+    buffer.getUint8(offset + 3)
+  );
+}
+
+export function ReadTwoCharCode(buffer: DataView, offset: number): string {
+  return String.fromCharCode(
+    buffer.getUint8(offset + 0),
+    buffer.getUint8(offset + 1)
+  );
+}
+
+export function ReadInt32(buffer: DataView, offset: number): number {
```

Comment:
Because I'm basically "copying and pasting" from the older CalChart code base, I'd like to try to keep these as similar as possible, so having the Read32 be a subroutine actually helps.

Also, I can then add logging when needed.  So I'm going to keep it as a separate function.